BUILD.bazel

```diff
@@ -1358,6 +1358,9 @@ PROD_AVX512VBMI_MICROKERNEL_SRCS = [
     "src/x8-lut/gen/x8-lut-avx512vbmi-vpermx2b-u128.c",
 ]
 
+PROD_AVX512VNNI_MICROKERNEL_SRCS = [
```

Comment:
Should we add generated avx512vnni sources here? Such as
"src/qs8-gemm/gen/qs8-gemm-1x16c4-minmax-fp32-avx512vnni.c",
"src/qs8-gemm/gen/qs8-gemm-4x16c4-minmax-fp32-avx512vnni.c",